woocommerce/woocommerce-ios
# My Store Analytics: Move the granularity text below the x-axis on the custom range tab

Closes: #12290 


## Description

This PR updates the position of the granularity text on the custom range tab of the My Store Analytics section. Joe suggested to keep the text subtle and left-aligned, but I think the text should have the same font and color as the other titles in the chart, and center aligned to look better. The discussion is ongoing here [p1710750371417729/1709651270.092029-slack-C0354HSNUJH] - I'm keeping the UI my preferred way for now. 

## Testing instructions

- Log in to the app.
- Create a new custom range or switch to the custom range tab on the My Store screen if you added one to your store already.
- Confirm that the granularity label is now below the x-axis and not hidden when selecting any time range on the chart.
- Also confirm that the granularity label is not displayed on other tabs.

## Screenshots

Custom range tab | Other tabs 
--- | ---
<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/c19e95d4-e6b8-4eec-b65a-1251d338f562" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/40e9c3ea-c659-4e58-8d20-1f1ab32f39f4" width=320 />


---
